### ISSUE TITLE
Bug issue with some Records counting

### ISSUE THREAD

**Issue by evillman** [odota/web#1790]
**Current Behavior**
Checking records web page (https://www.opendota.com/players/30345330/records/neutral_kills), it show more than 69000 neutral creep kills in a single 49min match. Which is impossible.

**Expected behavior/code**
Count correct number of neutrals killed.

**Possible Solution**
Re-parse game?

**Comment by coreymaher** [comment]
A few patches ago there was an issue with how those numbers were calculated. It has since been fixed but the bad data remains unless a reparse was requested. Unfortunately the replays are most likely long gone now, so the data cannot be fixed now...

**Comment by howardchung** [comment]
Technically we could delete all the data for matches in that time range, but I haven't done this yet

**Comment by Argn0** [comment]
maybe we could filter out implausible numbers client side?

**Comment by howardchung** [comment]
Yeah, although we should really avoid putting that kind of manual filtering in the web UI.
